### ISSUE TITLE
SIGBUS error after upgrading to v6.1.0 and webpack v5

### ISSUE THREAD

**Issue by malthejorgensen** [TypeStrong/fork-ts-checker-webpack-plugin#562]
Note that this is a stub issue, and will be updated with more info later. I can reliably reproduce this in our production repository, but it will take me some to create a minimal repository that reliably reproduces the issue. I'm creating the issue to hear if other people are experiencing the same.
Some of my coworkers are getting the same error, so it's not specific to my computer or Node.js setup.

## Current behavior

After upgrading from webpack v4 to v5 and upgrading fork-ts-checker-webpack-plugin from v4.1.6 to v6.1.0 (since v4 isn't compatible with webpack v5) I've been getting SIGBUS errors during webpack watch quite frequently:

```
> env TSC_WATCHFILE=UseFsEventsWithFallbackDynamicPolling npm run start -C js

[... bunch of normal output ...]
```

I then run a script that generates some type definintions from my GraphQL schema, and when I do that I get a SIGBUS error:

```
fish: 'env TSC_WATCHFILE=UseFsEventsWi…' terminated by signal SIGBUS (Misaligned address error)
(node:19591) UnhandledPromiseRejectionWarning: RpcIpcMessagePortClosedError: Cannot send the message - the message port has been closed for the process 19591.
    at /Users/malthe/eduflow/js/node_modules/fork-ts-checker-webpack-plugin/lib/rpc/rpc-ipc/RpcIpcMessagePort.js:47:47
    at processTicksAndRejections (internal/process/task_queues.js:83:21)
(node:19591) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 1)
(node:19591) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
(node:19591) UnhandledPromiseRejectionWarning: RpcIpcMessagePortClosedError: Cannot send the message - the message port has been closed for the process 19591.
    at /Users/malthe/eduflow/js/node_modules/fork-ts-checker-webpack-plugin/lib/rpc/rpc-ipc/RpcIpcMessagePort.js:47:47
    at processTicksAndRejections (internal/process/task_queues.js:83:21)
(node:19591) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 2)
```

## Expected behavior
Before the package upgrade I did not get any SIGBUS errors when outputting the generated files.
This happens on both node v12.20.1 and v14.5.0, and it happens both when using `env TSC_WATCHFILE=UseFsEventsWithFallbackDynamicPolling` and when not using that. When I omit the `TSC_WATCHFILE=...` I don't get the full traceback, I just get the SIGBUS error.

## Steps to reproduce the issue

Run webpack in watch mode, and then modify/add/remove some of the watched files.

## Issue reproduction repository

... coming in a couple of days ...

## Environment
 - **fork-ts-checker-webpack-plugin**:  6.1.0 
 - **typescript**: 4.1.2
 - **eslint**: 7.16.0
 - **webpack**: 5.13.0
 - **os**: macOS 10.15.7

**Comment by piotr-oles** [comment]
Have you tried to increase the memory limit?

**Comment by malthejorgensen** [comment]
I did the following change to my Webpack config – but I'm still getting the SIGBUS error:

```diff
- new ForkTsCheckerWebpackPlugin({})
+ new ForkTsCheckerWebpackPlugin({ typescript: { memoryLimit: 4096 } })
```

**Comment by piotr-oles** [comment]
And could you check if `tsc --watch --noEmit` works?

**Comment by malthejorgensen** [comment]
I am unable to get `tsc --watch --noEmit` to crash, whereas I can get webpack to crash quite easily. What I do is:

```bash
[ starting on `master` branch ]
> git checkout branch-with-substantial-changes
> npm run relay
[ other terminal with webpack crashes with SIGBUS ]
```

repeating these same steps does not crash `tsc --watch --noEmit` (tried 6-7 times).

I will try and get a repo with a minimal reproduction case set up this weekend, if I have time.

**Comment by piotr-oles** [comment]
Thanks! I'm going on vacation for next 2 weeks so I will be able to take a look at this only in 3 weeks :)

**Comment by piotr-oles** [comment]
Any updates? :)

**Comment by malthejorgensen** [comment]
Hey, I've tried to create a repo which could reproduce the issue, but I'm still unable to.

Our relay command works roughly like this:

    <build relay types into temporary dir> && rimraf __generated__ && <copy relay types to __generated>

One important bit of information that we found out is that switching out `rimraf` for regular `rm -rf` stopped the issue.

The `__generated__` directory contains 700 files totalling just over 4MB. My test repo basically recreated that setup (700 files / 4MB + rimraf + fork-ts-checker-webpack-plugin v6.1.0) but still no `SIGBUS`.

My production repo is still able to show the issue. I'll try and change tactics, to boil down the production repo to a minimal reproducing case.
